scripts/actions/spells/trust/rahal.lua

```diff
@@ -12,7 +12,30 @@ spellObject.onSpellCast = function(caster, target, spell)
 end
 
 spellObject.onMobSpawn = function(mob)
-    xi.trust.message(mob, xi.trust.messageOffset.SPAWN)
+    xi.trust.teamworkMessage(mob, {
```

Comment:
This trust has more chat lines than this: https://ffxiclopedia.fandom.com/wiki/Trust:_Rahal